sysutils/libvirt/Portfile

```diff
@@ -135,15 +134,27 @@ variant libssh2 description {Enable the libssh2 transport} {
     configure.args-replace  --without-ssh2 --with-ssh2
 }
 
-variant phyp requires libssh2 description {Phyp driver support} {
-    configure.args-replace  --without-phyp --with-phyp
-}
-
 variant sasl description {Use Cyrus SASL for authentication} {
     depends_lib-append      port:cyrus-sasl2
     configure.args-replace  --without-sasl --with-sasl
 }
 
+variant qemu description {Local QEMU support} {
+    depends_lib-append      port:qemu
+    configure.args-replace  --without-qemu --with-qemu
+
+    add_users               libvirt group=libvirt realname=${description} home=${prefix}/var/lib/libvirt shell=/sbin/nologin
+
+    configure.args-append   --with-qemu-user=qemu \
+                            --with-qemu-group=admin
+
+    startupitem.create      yes
+    startupitem.name        ${name}
+    startupitem.pidfile     ${prefix}/var/run/libvirtd.pid
+    startupitem.start       ${prefix}/sbin/libvirtd
```

Comment:
It's unusual for only one variant of a port to require a startupitem but if that's really the case here then it's fine.

If there's any way to make it use `startupitem.executable` instead of `startupitem.start` and `startupitem.pidfile` then that would be preferred. For example, does the `--daemon` flag help? (I haven't tried it.)

Reply:
I removed the `startupitem` for now; it isn't all that useful, as the user session works just fine.